src/components/presentational/SimpleCalendar/SimpleCalendar.css

```diff
@@ -0,0 +1,35 @@
+.mdhui-simple-calendar {
+    background: #fff;
```

Comment:
Would remove background; it is up to the card or container what the background is typically...and if we really need a contrast we would want to use the css background vars anyways so it works in dark mode

Reply:
This is gone now as part of the PR revisions.